cluster-autoscaler/core/scaleup/orchestrator/orchestrator.go

```diff
@@ -89,6 +89,7 @@ func (o *ScaleUpOrchestrator) ScaleUp(
 	nodes []*apiv1.Node,
 	daemonSets []*appsv1.DaemonSet,
 	nodeInfos map[string]*schedulerframework.NodeInfo,
+	allOrNothing bool, // Either request enough capacity for all unschedulablePods, or don't request it at all.
```

Comment:
nit: Could you also copy the comment to the Orchestrator interface (e.g. in the follow-up PR)?

Reply:
Will do in follow-up PR